cmd/kubeadm/app/node/BUILD

```diff
@@ -33,3 +33,17 @@ go_library(
         "//vendor:github.com/square/go-jose",
     ],
 )
+
+go_test(
+    name = "go_default_test",
+    srcs = ["bootstrap_test.go"],
```

Comment:
This file doesn't exist so presubmits are failing. Can you run ./hack/update-bazel.sh with a clean workspace e.g. git stash then update bazel then git stash pop?